armi/tests/test_mpiParameters.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2019 TerraPower, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests of the MPI portion of the Parameters class."""
+from distutils.spawn import find_executable
+import unittest
+
+from armi import context
+from armi.reactor import composites
+from armi.reactor import parameters
+
+# determine if this is a parallel run, and MPI is installed
+MPI_EXE = None
+if find_executable("mpiexec.exe") is not None:
+    MPI_EXE = "mpiexec.exe"
+elif find_executable("mpiexec") is not None:
+    MPI_EXE = "mpiexec"
```

Comment:
I think the corresponding block of code that is in the `test_parameters()` file can be removed, right?

Reply:
Fixed.  Good call.  Sorry about that.